docs/range-tokens/deployment-tutorial.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Range Tokens Deployment Tutorial
+sidebar_label: Deployment Tutorial
+---
+
+The process to launch and manage a range token on UMA is surprisingly simple. This tutorial will show you how to deploy a range token using the Long Short Pair (LSP) contract and the UMA `launch-lsp` repository. By following these instructions, you will be able to launch a custom LSP on Kovan testnet and Ethereum mainnet.
+
+## Deployment process
+
+The following is an overview of the deployment steps below.
+
+1. Clone the repo
+2. Install the dependencies
+3. Set your parameters
+4. Run the deployment on testnet (Kovan)
+5. Run the deployment on mainnet (Ethereum)
+
+### Is your desired price identifier and collateral type supported?
+
+Before deploying a contract, you should verify that your desired price identifier and collateral currency are already approved on the network you are trying to deploy to.
+
+View approved price identifiers and currencies here:
+- [Approved mainnet collateral currencies](/uma-tokenholders/approved-collateral-currencies)
+- [Approved mainnet price identifiers](/uma-tokenholders/approved-price-identifiers)
+- [Approved Kovan collateral currencies subgraph query](https://thegraph.com/explorer/subgraph/umaprotocol/uma-kovan?query=Whitelisted%20Collateral%20Currencies)
+- [Approved Kovan price identifiers subgraph query](https://thegraph.com/explorer/subgraph/umaprotocol/uma-kovan?query=Pricefeed%20Identifiers)
+
+To add a new price identifier or collateral currency to mainnet, please propose one to UMA voters by following the instructions detailed [here](/uma-tokenholders/adding-price-id).
+
+To add a new price identifier or collateral currency to Kovan, please contact the UMA team in Discord.
+
+## Clone the repo
+
+1. Clone the repo:
+
+ - [launch-lsp](https://github.com/UMAprotocol/launch-lsp)
+```bash
+ git clone https://github.com/UMAprotocol/launch-lsp.git
+```
+## Install the dependencies
+
+You will need to install nodejs v14 (we recommend nvm to manage node versions) and yarn.
+
+Note: these additional dependencies are required -- you may or may not have them on your system already:
+
+- libudev
+- libusb
+
+These dependencies are installed on MacOS by installing the XCode Developer Tools. For Linux, the example ubuntu installation command for additional deps is:
+
+```bash
+sudo apt-get update && sudo apt-get install -y libudev-dev libusb-1.0-0-dev
+```
+## Setting your parameters
+
+Each deployment requires the following parameters to be set at the point of deployment. 
+
+### Mandatory deployment parameters
+
+- `gasprice:` The gas price used for your contract deployment.
```

Comment:
I think it'd be cool if we gave more context to this stuff. Right now it basically reads as a general lsp launch tutorial. Thoughts on adding a range token example? Could maybe use UMA's range token that recently expired, which would give some real world context to this.

Then you could talk about the fact that you need to use UMA as collateral and UMA/USD as the price identifier. Could also give a rough example of pricing after deployment. And also give some context to what setting the bounds means.

Reply:
Updated to include more context. Let me know if this isn't what you were looking for but kept it consistent with using the UMA range token as an example across all sections.